release-notes/mac/release-notes.md

```diff
@@ -1,3 +1,30 @@
+# v173 (June 17 - 2023)
+- Added support for mobile app install links.
```

Comment:
can you divide them into Features/Bug Fixes/Improvements? Separate headings for each

Reply:
sure.

Reply:
Paolo can do this